client/src/screens/Client/Home/components/modal/NewOrEditEvent.js

```diff
@@ -450,10 +486,12 @@ const NewOrEditEvent = ({
                 value={!!provider && provider.id}
                 onChange={handleProviderChange}
                 label="Provider"
+                defaultValue={providers[index]?.id}
```

Comment:
I didn't get the point of using `?` here. Can you explain?

Reply:
in ES/20 JavaScript we can handle null/undefined using '?'